common/test/acceptance/tests/discussion/test_cohort_management.py

```diff
@@ -954,35 +953,6 @@ def test_verify_that_deselecting_the_final_child_deselects_category(self):
         # category should not be selected.
         self.assertFalse(self.cohort_management_page.is_category_selected())
 
-    def test_verify_that_correct_subset_of_category_being_selected_after_save(self):
-        """
-        Scenario: Category should be selected on selecting final child.
-
-        Given I have a course with a cohort defined,
-        And a inline discussion with disabled Save button.
-        When I click on child topics
-        Then I see enabled saved button
-        When I select subset of category
-        And I click on save button
-        Then I see success message with
-        same sub-category being selected
-        """
-        self.cohort_discussion_topics_are_visible()
-
-        # enable some inline discussion topics.
-        self.cohort_management_page.select_cohort_some_inline_discussion()
-
-        # category should not be selected.
-        self.assertFalse(self.cohort_management_page.is_category_selected())
-
-        cohorted_topics_after = self.cohort_management_page.get_cohorted_topics_count(self.inline_key)
-
-        # verifies that changes saved successfully.
-        self.save_and_verify_discussion_topics(key=self.inline_key)
```

Comment:
This saves the change, no inline discussions were ever selected (so before and after are trivially equal).